modules/core/src/main/java/org/apache/ignite/internal/jdbc/thin/ConnectionProperties.java

```diff
@@ -560,4 +561,40 @@ public void setPartitionAwarenessPartitionDistributionsCacheSize(
      * @param qryEngine SQL Query engine name.
      */
     public void setQueryEngine(String qryEngine);
+
+    /**
+     * @return Transaction concurrency value.
+     */
+    public TransactionConcurrency getTransactionConcurrency();
+
+    /**
+     * Sets transaction concurrency.
+     *
+     * @param transactionConcurrency Transaction concurrecny.
+     */
+    public void setTransactionConcurrency(String transactionConcurrency);
+
+    /**
+     * @return Transaction timeout in milliseconds.
+     */
+    public int getTransactionTimeout();
+
+    /**
+     * Sets transaction timeout in milliseconds.
+     *
+     * @param transactionTimeout Transaction timeout in millicesonds.
+     */
+    public void setTransactionTimeout(int transactionTimeout) throws SQLException;
```

Comment:
let's use long, similar to thin client and ignite transactions.

Reply:
I use `int` because it uses in other properties of class, See - `getConnectionTimeout`, `getQueryTimeout`.